docs/components/Date.md

```diff
@@ -74,17 +77,25 @@ The format of the day, month and year inputs has been set to numbers only. This
 Below is a list of input cases that the date validation triggers an error for. As per the GDS, empty field errors will be shown first, followed by invalid field errors and then relational date errors.
 
 - **All visible inputs are empty and the date input is required (i.e. Required input property is set to true)** - Triggers when the day, month and year fields are all visible and the user leaves all the inputs empty. For example, the input "Day: , Month: , Year: ".
+  - **Error Message:** "Enter a Date"
```

Comment:
I'd probably not include these error messages here, as they could get out of sync with what's in the code quite easily.